package.json

```diff
@@ -84,7 +84,7 @@
     "raw-loader": "0.5.1",
     "reflect-metadata": "0.1.10",
     "remap-istanbul": "0.9.5",
-    "rxjs": "5.4.2",
```

Comment:
Hey @Blackbaud-MikitaYankouski, in your testing, did you run into any problems upgrading to this version of rxjs.  Our dependency chain isn't crystal clear, and we've ran into problems when upgrading the dependency in builder, but not upgrading it in the skyux2 components.

https://github.com/blackbaud/skyux2/blob/master/package.json#L108

Reply:
Just re-ran the all the tests and haven't ran into any issues!

Reply:
The template also works with these changes. A point of note: UX does not directly depend on rxjs, so the mixmatched versions shouldn't affect a SPA.

Reply:
We are using a third party library that is required by our app that is more heavily dependent on rxjs, and they use this higher version and having two version in our node_modules gave us build errors.